pt-BR/mail/chrome/messenger/preferences/downloads.dtd

```diff
@@ -1,24 +1,24 @@
-<!ENTITY askOnSave.label              "Attachments Folder">
-<!ENTITY alwaysAsk.label              "Ask me where to save every file">
+<!ENTITY askOnSave.label              "Pasta de anexos">
+<!ENTITY alwaysAsk.label              "Perguntar onde salvar cada arquivo">
 <!ENTITY alwaysAsk.accesskey          "A">
-<!ENTITY neverAsk.label               "Save all attachments to this folder:">
+<!ENTITY neverAsk.label               "Salvar todos os anexos nesta pasta:">
 <!ENTITY neverAsk.accesskey           "S">
-<!ENTITY desktop.label                "Desktop">
-<!ENTITY downloads.label              "My Downloads">
-<!ENTITY chooseFolderWin.label        "Browse...">
+<!ENTITY desktop.label                "Área de Trabalho">
+<!ENTITY downloads.label              "Downloads">
+<!ENTITY chooseFolderWin.label        "Procurar...">
 <!ENTITY chooseFolderWin.accesskey    "B">
-<!ENTITY chooseFolderMac.label        "Choose...">
+<!ENTITY chooseFolderMac.label        "Escolher...">
 <!ENTITY chooseFolderMac.accesskey    "C">
 
 
-<!ENTITY fileTypes.label              "Download Actions">
-<!ENTITY configureActions.label       "View &amp; Edit Actions...">
+<!ENTITY fileTypes.label              "Ações de Download">
+<!ENTITY configureActions.label       "Ações Exibir e Editar...">
 <!ENTITY configureActions.accesskey   "V">
-<!ENTITY fileTypesDescription.label   "&brandShortName; can automatically save or open attachments of certain types.">
+<!ENTITY fileTypesDescription.label   "O &brandShortName; pode salvar ou abrir anexos de alguns tipos automaticamente.">
 
-<!ENTITY fileTypeColName.label        "File Type">
-<!ENTITY fileHandlerColName.label     "Application">
-<!ENTITY editFileHandler.label        "Change Action...">
+<!ENTITY fileTypeColName.label        "Tipo do Arquivo">
+<!ENTITY fileHandlerColName.label     "Aplicação">
+<!ENTITY editFileHandler.label        "Alterar Ação...">
 <!ENTITY editFileHandler.accesskey    "a">
-<!ENTITY removeFileHandler.label      "Remove">
+<!ENTITY removeFileHandler.label      "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY removeFileHandler.label      "Excluir">